otter/test/convergence/test_gathering.py

```diff
@@ -419,145 +424,128 @@ def test_invalid_first_entry(self):
         self.assertRaises(ValueError, extract_CLB_drained_at, feed)
 
 
+def lb_req(method, url, json_response, response):
```

Comment:
And while I'm doing that, I'll also always add on `loadbalancers` as the first part of the URL.